src/components/Navigation.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Logo from '../assets/planet.png';
+
+const Navigation = () => {
+  const navLinkStyles = ({ isActive }) => ({
+    textDecoration: isActive ? 'underline' : 'none',
+  });
+  return (
+    <div className="nav-container">
+      <div className="navs">
+        <img className="logo" src={Logo} width="50" height="50" alt="logo" />
+        <h2>Space Traveler&apos; s Hub</h2>
+      </div>
+      <div>
+        <NavLink className="nav-links" style={navLinkStyles} to="/">Rockets</NavLink>
+        <NavLink className="nav-links" style={navLinkStyles} to="/messions">Messions</NavLink>
```

Comment:
- [**OPTIONAL** ] I think you mean `Missions` instead of `Messions`, right?  So in my opinion it would be more professional to fix the typo. 🤓